# 订阅 sensor_msgs/JointState 消息
std_msgs/Header header 
string[] leftArmName
string[] rightArmName
float64[] leftArmPosition 
float64[] rightArmPosition 
float64[] leftArmVelocity
float64[] rightArmVelocity
